client/components/deposits-overview/index.tsx

```diff
@@ -39,19 +39,12 @@ const DepositsOverviewDetails: React.FunctionComponent< OverviewProps > = ( {
 };
 
 const DepositsOverview = (): JSX.Element => {
-	const {
-		overviews,
-		isLoading,
-	} = useAllDepositsOverviews() as AccountOverview.OverviewsResponse;
-
-	const { currencies, account } = overviews;
+	const { account, overview, isLoading } = useSelectedCurrencyOverview();
```

Comment:
This new hook will return the required logic to return the deposits overview for the selected currency only.